src/components/section/Section.tsx

```diff
@@ -24,7 +25,12 @@ export function Section(props: IProps) {
       <SectionHeading text={sectionTexts} fallback={props.section.beskrivendeId} />
 
       {props.section?.fakta?.map((faktum) => (
```

Comment:
Forslag! Siden det er tre props her, kanskje det er penere og mer lestbart med ´const { section, readonly, showAllFaktumTexts } = props;`? Tillegg, du har brukt denne fremgangsmåten alle andre steder

Reply:
Når komponentene er så små så pleier jeg ikke å bruke destructure fordi jeg ikke synes det gir noe særlig. Derimot når komponentene blir større og props repeteres mye så hender det jeg bruker det. 
Ofte synes jeg det faktisk er mer lesbart å bruke "props." fordi det sier meg med en gang at dette er data som kommer via props istedet for noe som bare er lokalt i komponenten.